assets/js/qa.js

```diff
@@ -0,0 +1,47 @@
+var currentQuestion = 0;
+var score = 0;
+var totQuestions = questions.length;
+
+
+var container =document.getElementById("quizContainer");
```

Comment:
The original doesn't have a capital `C`, so it should be the same here:

```suggestion
var container =document.getElementById("quizcontainer");
```